MAINTAINING.md

```diff
@@ -0,0 +1,23 @@
+# Maintaining UMA Repositories
+
+As our repos grow in activity, it takes more and more effort to maintain them and ensure code and issues are being
+actively triaged for consideration and review. This document outlines the responsibilities of the on-duty maintainer.
+
+## Responsibilities
+
+- Every business day, the on-duty maintainer is expected to make a pass over _all_ open PRs and new issues in _all_
+  UMA repositories.
+- For each PR, the on-duty maintainer should make sure the PR is moving forward:
```

Comment:
It seems like the ENG team being assigned to PRs and making sure they are continually moving forward is time consuming and takes away from items in the existing sprint. Would it be useful to track the work ENG is doing on these PRs in our existing sprint?

Reply:
It definitely takes time, but I'm not sure how useful it will be to explicitly track the review work in the sprint. I'm definitely open to the idea, though -- what sort of tracking process would you propose?